ext/extconf.rb

```diff
@@ -83,10 +83,9 @@ def safe_sh(cmd)
 
 # Absolutely prevent the linker from picking up any other zookeeper_mt
 Dir.chdir("#{HERE}/lib") do
-  %w[st mt].each do |stmt|
```

Comment:
Why remove the line copying the `mt` version?

Reply:
based on https://github.com/zk-ruby/zookeeper/pull/86/files#diff-b3c266c0a40fd6ea5ce51093b147dbe6R93 there is no `mt` usage. I could split into 2 commits with dylib adding and then with small refactoring

Reply:
I think, yep I'll create separate PR for that refactoring

Reply:
Updated PR without this change